app/uk/gov/hmrc/hmrcemailrenderer/templates/mandate/mandateAgentSelfAuthAdds.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, params("service").toString + " update") {
+
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">You have successfully added your client, you now have the authority to act for @{params("service").toString}.
+
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">From HMRC</p>
```

Comment:
It has "From HMRC" in the bottom...which is same as we use in rest of the mandate templates